ubuntu_image/classic_builder.py

```diff
@@ -192,6 +192,12 @@ def populate_bootfs_contents(self):
             self._populate_one_bootfs(name, volume)
         super().populate_bootfs_contents()
 
+    def make_disk(self):
+        super().make_disk()
+        user = os.environ['SUDO_USER']
```

Comment:
If you are calling a program under sudo, it is expected that its output will be owned by root.  While I see the convenience in having the output be owned by the non-root user instead, I am uncomfortable with "convenience" changes around security-sensitive interfaces like sudo.